website/cue/reference/components/sinks/loki.cue

```diff
@@ -219,6 +219,18 @@ components: sinks: loki: {
 				pod_labels_name: unicorn
 				"""
 		}
+
+	    request_encoding: {
+	        title: "Request Encoding"
+	        body: """
+	            Loki can receive log entries as either snappy-compressed
+	            protobuf or JSON. If the request is sent as JSON, the
+	            request may also be gzipped.
+
+	            For the `loki` sink this means the body will be encoded based
+	            on the configured `compression`.
```

Comment:
Nit but the beginning of the sentence sounds slightly redundant since we are already within the context of the loki sink.

```suggestion
	            The protobuf body will be encoded based on the configured `compression`.
```

Reply:
Don't disagree, was following the "local" behavior: https://vector.dev/docs/reference/configuration/sinks/loki/#event-ordering

Reply:
Feel free to drop this- it was just something I noticed but, I don't yet have a good grasp on external doc writing correctness.

Reply:
TBF, we don't have consistency in writing or style - so it's a very valid observation, and as a whole - I don't disagree 😄